### ISSUE TITLE
[Feature request] File transfer between workspaces

### ISSUE THREAD

**Issue by nemanjan00** [PeterCxy/Shelter#38]
For a lot of apps, just share to another workspace is not enough, some of them need to be able to open the file... 

It would be cool if there was something like [Save To File](https://play.google.com/store/apps/details?id=io.xjhub.savetofile) built in.

**Comment by PeterCxy** [comment]
Does `Save To File` work with Shelter currently?

**Comment by nemanjan00** [comment]
Yes, it does. I just think it would be nice to have it built in :D
